feature-detail/src/androidMain/kotlin/detail/data/DetailStreamService.kt

```diff
@@ -1,9 +1,9 @@
-package com.codandotv.streamplayerapp.feature_list_streams.detail.data
+package detail.data
```

Comment:
The same point I said before, I think it is something going wrong with our name space for this module